RegistryManager/src/test/java/eu/neclab/ngsildbroker/registryhandler/controller/RegistryControllerTest.java

```diff
@@ -0,0 +1,303 @@
+package eu.neclab.ngsildbroker.registryhandler.controller;
+
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.Response.Status;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import eu.neclab.ngsildbroker.commons.constants.AppConstants;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+
+@QuarkusTest
+@TestMethodOrder(OrderAnnotation.class)
+@TestProfile(CustomProfile.class)
+public class RegistryControllerTest {
+
+	private String payload;
+	private String payload1;
```

Comment:
Please give the name related to test case.

Reply:
done